txpool/src/config.rs

```diff
@@ -0,0 +1,16 @@
+#[derive(Debug, Clone)]
+pub struct Config {
+    /// Maximum number of transaction inside pool
+    pub max_tx: usize,
+    /// max depth of connected UTXO excluding contracts
+    pub max_depth: usize,
+}
+
+impl Default for Config {
+    fn default() -> Self {
+        Self {
+            max_tx: 4064,
+            max_depth: 10,
```

Comment:
Hmmm. Did we not say this should be `1`? Or is this something else?

Reply:
We talked if it is easier to do it with just depth 1, but it was more easier to do it with depth counting (logic is few lines).